input/otlp/traces.go

```diff
@@ -450,11 +461,6 @@ func TranslateTransaction(
 			case semconv.AttributeRPCService:
 			case semconv.AttributeRPCMethod:
 
-			// URL
```

Comment:
Note for reviewer: The `url.full` attribute is not expected on spans for HTTP server requests, so I have removed this.